### DISCUSSION TITLE
guidance for contribution

### DISCUSSION THREAD

**Discussion by gaju91** [socketio/socket.io#3652] [General]
I am new to programming and want to contribute , can you guide me about what should I learn in order to that ?

**Comment by darrachequesne** [comment]
Hi! I think the best way to start is to create something with Socket.IO. Any application will work, from the classic chat to a simple multiplayer game.

Once you've done that, you should have a better overview of the use cases covered by the project, and the things that are missing/could be improved.

If you encounter any issue, you can join the Slack channel: https://socketio-slackin.herokuapp.com/

**Reply by gaju91** [reply]
thanks i will follow that  and will do my best to make socket.io better
